api/v1/users/PKPUserHandler.inc.php

```diff
@@ -279,4 +284,60 @@ private function _processAllowedparams($params, $allowedKeys) {
 
 		return $returnParams;
 	}
+
+	/**
+	 * Retrieve the user report
+	 *
+	 * @param Slim\Http\Request $slimRequest Slim request object
+	 * @param \APIResponse $response Response
+	 * @param array $args
+	 * @return \APIResponse Response
+	 */
+	public function getReport(\Slim\Http\Request $slimRequest, \APIResponse $response, array $args): \APIResponse
+	{
+		$request = $this->getRequest();
+
+		$context = $request->getContext();
+		if (!$context) {
+			return $response->withStatus(404)->withJsonError('api.404.resourceNotFound');
+		}
+
+		$params = [
+			'contextId' => $context->getId(),
+			'serializer' => \PKP\User\Report\Serializer\CSV::class
```

Comment:
Just left it here in case there's interest to add other formats, if not, I can remove the `abstraction` =]